apps/check-processing-time/check-processing-time-cf.yml.j2

```diff
@@ -0,0 +1,84 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+
+  PermissionsBoundaryPolicyArn:
+    Type: String
+    Default: ""
```

Comment:
We've avoided setting default parameters anywhere outside of `main.cf.j2`. In the past it's helped catch errors where the parent stack wasn't correctly passing a value to a child stack.
```suggestion
```

Reply:
Makes sense, I also removed that line from the `get-files` template (which is where I originally copied it from).